content/guides/user-guides/delfin/driver-matrix-support/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Delfin Driver Support Matrix
+menuTitle: Driver Support Matrix
+description: "Delfin Driver Support Matrix"
+weight: 40
+disableToc: false
+tags: ["user guide", "delfin", "drivers"]
+---
+
+The following drivers are supported by delfin.
+
+{{%children style="h3" description="true" %}} 
+
+If you want to add new delfin driver support matrix, find the link [here](https://docs.sodafoundation.io/guides/user-guides/delfin/add-driver-support/) 
```

Comment:
Please add index to all supported drivers.. use the same template for all

Reply:
done